backend/scripts/location_differ.py

```diff
@@ -244,7 +348,7 @@ def location_differ(
             lat, lng = get_coordinates(
                 title=geojson["properties"]["name"],
                 subtitle=geojson["properties"]["address"],
-                api_key=api_key,
+                api=gmaps,
```

Comment:
do we need the temporary?

Reply:
We do since the `geojson` is a prelim object